extensions.json

```
@@ -1047,6 +1059,11 @@
       "id": "twxs.cmake",
       "repository": "https://github.com/twxs/vs.language.cmake"
     },
+    {
+      "id": "vhdlwhiz.vhdl-by-vhdlwhiz",
+      "repository": "https://github.com/jonasjj/awesome-vhdl",
+      "version": "1.2.4"
```

```suggestion
      "version": "1.2.4",
      "checkout": "v1.2.4"
```

This is to ensure that we build exactly from the sources of the 1.2.4 version.